vault/logical_system_paths.go

```diff
@@ -1969,9 +2428,111 @@ func (b *SystemBackend) mountPaths() []*framework.Path {
 				},
 			},
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   b.handleMountTuneRead,
-				logical.UpdateOperation: b.handleMountTuneWrite,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: b.handleMountTuneRead,
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"max_lease_ttl": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["tune_max_lease_ttl"][0]),
+									Required:    true,
+								},
+								"description": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["auth_desc"][0]),
+									Required:    true,
+								},
+								"default_lease_ttl": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["tune_default_lease_ttl"][0]),
+									Required:    true,
+								},
+								"force_no_cache": {
+									Type:     framework.TypeBool,
+									Required: true,
+								},
+								"token_type": {
+									Type:        framework.TypeString,
+									Description: strings.TrimSpace(sysHelp["token_type"][0]),
+									Required:    false,
+								},
+								"allowed_managed_keys": {
+									Type:        framework.TypeCommaStringSlice,
+									Description: strings.TrimSpace(sysHelp["tune_allowed_managed_keys"][0]),
+									Required:    false,
+								},
+								"allowed_response_headers": {
+									Type:        framework.TypeCommaStringSlice,
+									Description: strings.TrimSpace(sysHelp["allowed_response_headers"][0]),
+									Required:    false,
+								},
+								"allowed_request_headers": {
```

Comment:
where do you see the field `allowed_request_headers`?

Reply:
should have been `"passthrough_request_headers"` fixed now.

Reply:
ah nvm that was already there... yeah idk how I added that but removed now.